front/app/containers/Admin/messaging/AutomatedEmails/ExampleModal/index.tsx

```diff
@@ -67,63 +46,28 @@ const ExampleModal = ({
     );
   };
 
-  if (isNilOrError(examples) || !campaign) return null;
+  if (isNilOrError(examples)) return null;
 
   const selectedExample =
-    selectedExampleIdx === null ? null : examples.data[selectedExampleIdx];
-
-  // New preview only works with editing enabled & where the BE has a preview class
-  // TODO: Seems to call /preview twice
-  const hasPreview = isEditingEnabled && campaign.data.attributes.has_preview;
-
-  const handleSendPreviewEmail = () => {
-    sendCampaignPreview(campaignId, {
-      onSuccess: () => {
-        const previewSentConfirmation = formatMessage(
-          messages.previewSentConfirmation
-        );
-        window.alert(previewSentConfirmation);
-      },
-    });
-  };
+    // TODO: Fix this the next time the file is edited.
+    // eslint-disable-next-line @typescript-eslint/no-unnecessary-condition
+    selectedExampleIdx === null ? null : examples?.data[selectedExampleIdx];
```

Comment:
```suggestion
    selectedExampleIdx === null ? null : examples.data[selectedExampleIdx];
```

and then you don't need to disable eslint